lib/Common/Memory/Recycler.h

```diff
@@ -1502,7 +1502,7 @@ class Recycler
     template <typename SmallHeapBlockAllocatorType>
     void RemoveSmallAllocator(SmallHeapBlockAllocatorType * allocator, size_t sizeCat);
     template <ObjectInfoBits attributes, typename SmallHeapBlockAllocatorType>
-    char * SmallAllocatorAlloc(SmallHeapBlockAllocatorType * allocator, size_t sizeCat);
+    char * SmallAllocatorAlloc(SmallHeapBlockAllocatorType * allocator, size_t sizeCat, size_t size);
```

Comment:
What is the perf impact of taking in an additional argument? Please do a perf run to make sure that non-page heap builds are not impacted